src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -69,7 +69,7 @@ limitations under the License.
     <item name="ApplicationID" type="char_string"/>
   </struct>
 
-  <enum name="ApplicationLauncherStatusEnum" type="enum8">
+  <enum name="StatusEnum" type="enum8">
```

Comment:
Was there a reason a backwards compat shim for the old name was not added?

Reply:
created https://github.com/project-chip/connectedhomeip/pull/30202